lerna.json

```diff
@@ -1,10 +1,10 @@
 {
   "npmClient": "yarn",
   "version": "independant",
-  "useWorkspaces": true,
```

Comment:
This is now the default value and was recommended to be removed via `lerna repair`